testsuite/tests/kuadrant/authorino/operator/http/conftest.py

```diff
@@ -0,0 +1,48 @@
+"""Conftest for all tests requiring custom deployment of Authorino"""
+import pytest
+from weakget import weakget
+
+from testsuite.openshift.objects.authorino import AuthorinoCR
+from testsuite.objects import Authorino, Authorization
+from testsuite.openshift.objects.auth_config import AuthConfig
+from testsuite.httpx import HttpxBackoffClient
+
+HTTP_DEFAULT_PORT=5001
```

Comment:
I believe the services have their ports named so you could expose the route and use a name instead an actual port